services/ui/src/components/SshKeys/AddSshKey.js

```diff
@@ -0,0 +1,123 @@
+import React, { useState } from 'react';
+import css from 'styled-jsx/css';
+import Button from 'components/Button';
+import { Mutation } from 'react-apollo';
+import Me from 'lib/query/Me';
+import { bp, color, fontSize } from 'lib/variables';
+import AddSshKeyMutation from '../../lib/mutation/AddSshKey';
+
+const AddSshKey = ({me: { id, email }}) => {
+
+  const defaultValues = {sshKeyName: '', sshKey: '', sshKeyType: 'SSH_RSA'};
+  const [values, setValues] = useState(defaultValues);
+
+  const handleChange = e => {
+    const {name, value} = e.target;
+    setValues({...values, [name]: value});
+  }
+
+  const isFormValid = values.sshKeyName !== '' && values.sshKey !== '';
+
+  return(
+    <div className="addSshKey">
+
+      <Mutation mutation={AddSshKeyMutation} refetchQueries={[{ query: Me }]}>
+        {(addSshKey, { loading, called, error, data }) => {
+
+          const addSshKeyHandler = () => { 
+            addSshKey({
+              variables: {
+                input: {
+                  name: values.sshKeyName,
+                  keyValue: values.sshKey.replace('ssh-rsa', '').replace('ssh-ed25519', '').trim(),
+                  keyType: values.sshKeyType,
+                  user: {
+                    id,
+                    email
+                  }
+                }
+              }
+            });
+            setValues(defaultValues);
+          };
+
+          if (!error && called && loading) {
+            return <div>Adding SSH Key...</div>;
+          }
+
+          return (
+            <div className="addNew">
+
+              { error ? <div className="error">{error.message.replace('GraphQL error:', '').trim()}</div> : "" } 
+
+              <div>
+                <label>SSH Key Name</label>
```

Comment:
Is there a reason there's no `htmlFor` attribute here? Accessibility is covered by `aria-labelledby` but I think this label wouldn't be click-selectable.

Reply:
it's added for `htmlFor="sshKeyType"` so I wasn't sure if this was intentionally left off

Reply:
and, now that i'm looking closely, the aria label `addSshKeyName` doesn't match any IDs that I see, so that isn't working as intended either (I think)